src/libstd/build.rs

```diff
@@ -1,9 +1,7 @@
 #![deny(warnings)]
 
-use std::env;
-
 fn main() {
-    let target = env::var("TARGET").expect("TARGET was not set");
+    let target = env!("TARGET","TARGET was not set");
```

Comment:
I don't believe that `env!` takes multiple arguments; how did you test this?

Reply:
it does https://doc.rust-lang.org/std/macro.env.html#examples

Reply:
Looks like the [stub macro definitions](https://github.com/rust-lang/rust/blob/81970852e172c04322cbf8ba23effabeb491c83c/src/libstd/macros.rs#L504-L507) needs to be updated then.